gef-ui/server/controllers/about-facility/index.js

```diff
@@ -69,6 +70,7 @@ const validateAboutFacility = async (req, res) => {
 
   let coverStartDate = null;
   let coverEndDate = null;
+  let coverEndDateValid = true;
 
   if (isTrueSet(body.hasBeenIssued)) {
```

Comment:
just to highlight that this function is big. really big... - around 300 lines of code. there is a lot of validation going on. 
after launch we need to split it in small chunks

Reply:
left todo